demisto_sdk/commands/common/clients/xsoar/xsoar_api_client.py

```diff
@@ -980,6 +982,37 @@ def run_cli_command(
             investigation_id, response_type
         )
 
+    def get_formatted_error_entries(self, entries: List[Entry]) -> Set[str]:
+        """
+        Get formatted error entries from an executed command / playbook tasks
+
+        Args:
+            entries: a list of entries
+
+        Returns:
+            Formatted error entries
+        """
+        error_entries: Set[str] = set()
+
+        for entry in entries:
+            if entry.type == self._ENTRY_TYPE_ERROR and entry.parent_content:
+                # Checks for passwords and replaces them with "******"
+                parent_content = re.sub(
```

Comment:
what are you checking here? Why is the entry printing passwords? Is this the only case we need to check for?

Reply:
we talked about it previously [here](https://github.com/demisto/demisto-sdk/pull/3732#discussion_r1382542636)

Reply:
are there other forms of authentication or other secrets that might be printed?

Reply:
nope